pkg/kubectl/resource/builder.go

```diff
@@ -360,13 +396,33 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	switch {
 	case hasSlash > 0 && hasSlash == len(args):
 		return true, nil
-	case hasSlash > 0 && hasSlash != len(args):
-		return true, fmt.Errorf("when passing arguments in resource/name form, all arguments must include the resource")
```

Comment:
Maybe I'm wrong, but it seems the "find me the resource b in group a named c" behavior will only happen if there are multiple resources (see L322), and splitGroupResourceTypeName in L341 won't detect a problem that `c` doesn't contain the resource name.

It would be great if you can run kubectl to confirm if it's correct. Thanks!